### PR TITLE
fix: homepage and verification auth icons

### PR DESCRIPTION
**Pull request by jessiiepeng**
## Implementation description. How did you make this change?
* Added numbers to desktop view

https://user-images.githubusercontent.com/53016529/151677730-d73192f7-50c8-493b-8cef-04d324a4080e.mov

* Added verification email images

https://user-images.githubusercontent.com/53016529/151677774-cef6e1a0-dfa7-4f9f-bd1f-9a6542405b7b.mov

[confirmed verification icon](https://drive.google.com/file/d/1710JqTk7lxUUBbvqG0S8Jqv_aiOPDL8i/view?usp=sharing)

* Added email verification expiry date

Screen Shot 2022-01-31 at 11 47 14 AM

## Steps to test
1. For donation step icons: test responsiveness with mobile/desktop and check it matches Figma
2. For verification email: check if it matches Figma by signing up for a new account
Note: ^ to test confirmed verification email page, change  in `App.tsx`, include `import Home from "./components/auth/Signup/ConfirmVerificationPage";`

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages follow conventional commits and are descriptive. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s) 
- [ ] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR
- [ ] The appropriate tests if necessary have been written

### CODE REVIEW

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -23,6 +23,7 @@ import customTheme from "./theme/index";
 import { AuthenticatedUser } from "./types/AuthTypes";
 import { getLocalStorageObj } from "./utils/LocalStorageUtils";
 
+
```
Comment (hanlinc27) [inline frontend/src/App.tsx:26]: revert the changes in this file?

File: frontend/src/components/auth/Signup/ConfirmVerificationPage.tsx [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
+import confirmVerificationImage from "../../../assets/authentication_complete.svg"
 
 import * as Routes from "../../../constants/Routes";
 
 const ConfirmVerificationPage = () => {
   const history = useHistory();
 
   return (
-    <Container centerContent pl="42px" pr="42px" pt="31px">
+    <Container pl="42px" pr="42px" pt="31px">
```
Comment (hanlinc27) [inline frontend/src/components/auth/Signup/ConfirmVerificationPage.tsx:13]: pass it the `dashboardContainer` variant?

File: frontend/src/theme/textStyles.ts [hunk 3]
```diff
@@ -128,5 +128,10 @@ const textStyles = {
     fontWeight: 500,
     lineHeight: "22px",
   },
+  iconNumber: {
```
Comment (hanlinc27) [inline frontend/src/theme/textStyles.ts:131]: nit: 
```suggestion
 desktopIconNumber: {
```

**Review by hanlinc27 — CHANGES_REQUESTED** [review]

File: frontend/src/theme/textStyles.ts (hunk 3 above)
Comment (jessiiepeng) [reply frontend/src/theme/textStyles.ts]: All the changes should be made!

File: frontend/src/components/auth/Signup/ConfirmVerificationPage.tsx [hunk 4]
```diff
@@ -1,15 +1,16 @@
...
 
+import confirmVerificationImage from "../../../assets/authentication_complete.svg";
 import * as Routes from "../../../constants/Routes";
 
 const ConfirmVerificationPage = () => {
   const history = useHistory();
 
   return (
-    <Container centerContent pl="42px" pr="42px" pt="31px">
+    <Container dashboardVariant>
```
Comment (hanlinc27) [inline frontend/src/components/auth/Signup/ConfirmVerificationPage.tsx:13]:
```suggestion
    <Container variant="dashboardVariant">
```

**Review by hanlinc27 — CHANGES_REQUESTED** [review]

**Review by hanlinc27 — APPROVED** [review]
image
